crates/weaver_resolver/src/attribute.rs

```diff
@@ -82,18 +82,22 @@ impl AttributeCatalog {
                 stability,
                 deprecated,
                 prefix,
+                rename,
             } => {
                 let name;
                 let root_attr = self.root_attributes.get(r#ref);
                 if let Some(root_attr) = root_attr {
                     let mut attr_lineage = AttributeLineage::new(&root_attr.group_id);
 
-                    if *prefix {
```

Comment:
fyi: prefix means that attribute will be embedded i.e. having parent prefix in his name